aries-site/src/themes/aries.js

```diff
@@ -15,7 +15,7 @@ export const aries = deepMerge(hpe, {
     },
     xsmall: {
       areas: [
-        ['parent', 'null'],
+        ['parent', 'parent'],
```

Comment:
I can't tag the proper place, but can we add:
```
actions: {
  pad: { vertical: "xxsmall" } // aligns button height with heading line-height
}
```

Reply:
Currently the top of the buttons align to the top of the headings line-height. For visual alignment, I think we should align to the font-height instead of the font's line-height. ☝️ this adjustment achieves that.

Reply:
Done.